run-android-studio-benchmark.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+AGP_VERSION="${AGP_VERSION:-7.1.0-beta01}"
+GRADLE_VERSION="${GRADLE_VERSION:-7.3.1}"
+KOTLIN_VERSION="${KOTLIN_VERSION:-1.4.30}"
+BENCHMARK="${BENCHMARK:-no-op-sync first-import-sync-scenario app-build-script-change-sync app-dependency-change-sync library-dependency-change-sync}"
+
+if [ -z ${ANDROID_STUDIO_INSTALL_DIR+x} ]; then
+    echo "Env. variable ANDROID_STUDIO_INSTALL_DIR should be set with the location of the Android Studio"
+    exit 1
+fi
+
+sync
+
+gradle-profiler/gradlew --project-dir gradle-profiler install
+
+gradle-profiler/build/install/gradle-profiler/bin/gradle-profiler \
```

Comment:
I wonder how much sense it makes to add the second file here. Wouldn't it be better to have one scenario file with all the scenarios in it? And one shell script which can execute a subset of the scenarios?

Reply:
As discussed this is good enough for now.